### PR TITLE
Typewriter7 API token update

### PR DESCRIPTION
**Pull request by vanand17**
### Proposed changes

Updated Typewriter7 API token docs to reflect that is only supports using Config API and not public API

### Merge timing
NA

### CODE REVIEW

File: src/protocols/apis-and-extensions/typewriter-v7.md [hunk 1]
```diff
@@ -114,13 +114,15 @@ $ npx typewriter@7
...
 You must be a workspace owner to create Segment API tokens.
 
+> note ""
+> Typewriter7 only uses Config API
+
+
 To create an API token:
 1. Click on the **Tokens** tab on the [Access Management](https://app.segment.com/goto-my-workspace/settings/access-management){:target="_blank"} page and click **Create Token**.
-2. Choose Segment's Public API.
+2. Choose Segment's Config API.
```
[the remark this answers is not in the document]
Comment (forstisabella) [reply src/protocols/apis-and-extensions/typewriter-v7.md:125]: It looks like users can no longer create Config API keys in the UI, according to #5824

[the remark this answers is not in the document]
Comment (vanand17) [reply src/protocols/apis-and-extensions/typewriter-v7.md:125]: Hey @forstisabella You should be able to select the "Looking for our original Config API?" and from there, you should be able to click Config API, and create a new one.

Let me know if this doesn't work, I can share a screenshot on Slack

Comment (forstisabella) [inline src/protocols/apis-and-extensions/typewriter-v7.md:125]:
```suggestion
2. Select Segment's Config API. __If you don't see this option, reach out to friends@segment.com for assistance.__
```

File: src/protocols/apis-and-extensions/typewriter-v7.md [hunk 2]
```diff
@@ -114,13 +114,15 @@ $ npx typewriter@7
 
 ## API Token Configuration
 
-Typewriter requires a Segment API token to fetch Tracking Plans from the [Segment Public API](/docs/api/public-api).
-
 You must be a workspace owner to create Segment API tokens.
 
+> note ""
+> Typewriter7 only uses Config API
```
Comment (forstisabella) [inline src/protocols/apis-and-extensions/typewriter-v7.md:120]:
```suggestion
> Typewriter7 only uses the Config API. 
```

File: src/protocols/apis-and-extensions/typewriter-v7.md [hunk 3]
```diff
@@ -114,13 +114,15 @@ $ npx typewriter@7
 
 ## API Token Configuration
 
-Typewriter requires a Segment API token to fetch Tracking Plans from the [Segment Public API](/docs/api/public-api).
-
 You must be a workspace owner to create Segment API tokens.
 
+> note ""
```
Comment (forstisabella) [inline src/protocols/apis-and-extensions/typewriter-v7.md:119]:
```suggestion
> info ""
```

File: src/protocols/apis-and-extensions/typewriter-v7.md [hunk 4]
```diff
@@ -114,13 +114,15 @@ $ npx typewriter@7
 
 ## API Token Configuration
 
-Typewriter requires a Segment API token to fetch Tracking Plans from the [Segment Public API](/docs/api/public-api).
-
 You must be a workspace owner to create Segment API tokens.
```
Comment (forstisabella) [inline src/protocols/apis-and-extensions/typewriter-v7.md:117]:
```suggestion
You must be a Workspace Owner to create Segment API tokens. For more information about roles in the Segment app, see the [Roles documentation](/docs/segment-app/iam/roles/). 
```

**Review by forstisabella — APPROVED** [review]

**Pull request merged**